_episodes/07-errors.md

```diff
@@ -394,10 +394,10 @@ One reason for receiving this error is that you specified an incorrect path to t
 For example,
 if I am currently in a folder called `myproject`,
 and I have a file in `myproject/writing/myfile.txt`,
-but I try to just open `myfile.txt`,
+but I try to open `myfile.txt`,
 this will fail.
 The correct path would be `writing/myfile.txt`.
-It is also possible (like with `NameError`) that you just made a typo.
+It is also possible (like with `NameError`) that the file path contains a typo.
```

Comment:
Move ```(like with `NameError`)``` to the end of the sentence, remove parentheses so that it reads something like this:

> It is also possible that the file name contains a typo in which case Python returns a `NameError`.

I'm not sure if we can you file _path_

Reply:
file path looks weird to me too but I do not know if using file name is precise enough (as here we have the path too).

Reply:
Regarding incorporating `NameError` without parentheses, if the file name contains a typo, it does not return a `NameError`, it is still a `FileNotFoundError`. I could drop the `NameError` reference for:
```
It is also possible that the file name or its path contains a typo.
```

Reply:
right... `NameError` is when there is a typo in a variable name. You can either drop these parentheses altogether or replace `NameError` with `FileNotFoundError` -- your call